.devcontainer/devcontainer.json

```diff
@@ -7,17 +7,19 @@
 	// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
 	"dockerFile": "Dockerfile",
 	// Set *default* container specific settings.json values on container create.
-	"settings": {
-		"terminal.integrated.shell.linux": null
+	"customizations": {
+		"vscode": {
+			"settings": {},
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"ms-python.python"
+			]
+		}
 	},
-	// Add the IDs of extensions you want installed when the container is created.
-	"extensions": [
-		"ms-python.python"
-	],
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
 	// Adding id_rsa so that we can push to github from the dev container
-	"initializeCommand": "ssh-add $HOME/.ssh/id_rsa",
+	"initializeCommand": "ssh-add",
```

Comment:
I think DM has seen this, but @mbh329 if you haven't, this is a more generic ssh-add that will add whatever credentials you have in .ssh

Reply:
I have not and thanks for flagging, should the comment above be updated as well?

Reply:
true. the one I wrote recently was 
```
# Add local SSH private keys in order to push to github from the dev container
```